risks/MASVS-STORAGE/1-store-sensitive-data-securely/data-unencrypted-external/android-data-unencrypted-external-static/example-1/example.md

```diff
@@ -0,0 +1,32 @@
+---
+platform: android
+title: Find common APIs that return paths to External Storage locations
```

Comment:
This example seems to be about Scoped Storage, maybe we can adapt the title and text to it. 


I created a new example that you can use for the non-scoped storage case: [output-demo-4.zip](https://github.com/OWASP/owasp-mastg/files/15370471/output-demo-4.zip)